WcaOnRails/spec/controllers/api_controller_spec.rb

```diff
@@ -227,20 +227,6 @@
       end
     end
 
-    context 'signed in as senior delegate' do
-      before :each do
-        api_sign_in_as(FactoryBot.create(:senior_delegate))
-      end
-
-      it 'has correct delegate_status' do
```

Comment:
Would it make sense to replace this test by an equivalent check?

Reply:
Hm, Rails should be smart enough to figure this out and cancel the loop during serialization.
Will do some more reading and get back to you!